packages/@fbcmobile-ui/Components/Core/ScrollViewWithBottomBar.js

```diff
@@ -24,14 +24,11 @@ type Props = {
 
 const ScrollViewWithBottomBar = (
   {children, bottomBar}: Props,
-  scrollViewRef: TRefFor<typeof ScrollView>,
```

Comment:
This is awesome! Could you elaborate here. How was this fixed?

Reply:
Flow picked up on it a while ago, but it had a $FlowFixMe in the other repo so it never popped up as a flow error.  It seems to not have been used anywhere, and the scrollviews seem to behave as expected without it.